pep-0626.rst

```diff
@@ -214,6 +214,13 @@ Note that these functions are not part of the C-API, so will be need to be linke
 
 ``PyLineTable_PreviousAddressRange`` retreats the range to the previous entry, returning non-zero if valid.
 
+.. note::
+  The data in ``linetable`` is immutable, but its lifetime depends on its code object.
+  For reliable operation, ``linetable`` should be copied into a local buffer before calling ``PyLineTable_InitAddressRange``.
+
+  Although these functions are not part of C-API, they will provided by all future versions of CPython.
```

Comment:
```suggestion
  Although these functions are not part of C-API, they will provided by all future versions of CPython and they will remain self-contained.
```

Reply:
What does "self contained" mean in this context? I've never heard that term applied to a C function.

Reply:
That it does not require the rest of the C-API (so copying all these functions to a different process will suffice).